### PR TITLE
ANSI 301-2022: load profile schedules

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Add tests for ANSI 301 schedules.

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] OS-HPXML git subtree has been pulled
- [ ] 301/ES rulesets and unit tests have been updated
- [ ] 301validator.xml has been updated (reference EPvalidator.xml)
- [ ] Workflow tests have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI

### CODE REVIEW

**Comment by joseph-robertson** [comment]
Are there any additional components that should be assigned constant monthly multipliers?
Are there any changes/updates that need to be made to the OS-ERI documentation?

**Comment by joseph-robertson** [comment]
Reminder to add schedule tests for the rest of the ANSI 301 schedules (recirc pump, lighting, etc).

File: rulesets/tests/test_occupancy.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative '../../hpxml-measures/HPXMLtoOpenStudio/resources/minitest_helper'
+require 'openstudio'
+require_relative '../main.rb'
+require 'fileutils'
+require_relative 'util.rb'
+
+class ERIApplianceTest < Minitest::Test
```
Comment (shorowit) [inline rulesets/tests/test_occupancy.rb:9]:
```suggestion
class ERIOccupancyTest < Minitest::Test
```

File: rulesets/tests/test_appliances.rb [hunk 2]
```diff
@@ -388,6 +394,9 @@ def _check_dishwasher(hpxml_bldg, ef:, annual_kwh:, cap:, elec_rate:, gas_rate:,
     assert_in_epsilon(gas_rate, dishwasher.label_gas_rate, 0.01)
     assert_in_epsilon(agc, dishwasher.label_annual_gas_cost, 0.01)
     assert_in_epsilon(label_usage, dishwasher.label_usage, 0.01)
+    assert_equal('0.015, 0.007, 0.005, 0.003, 0.003, 0.010, 0.020, 0.031, 0.058, 0.065, 0.056, 0.048, 0.042, 0.046, 0.036, 0.038, 0.038, 0.049, 0.087, 0.111, 0.090, 0.067, 0.044, 0.031', dishwasher.weekday_fractions)
+    assert_equal('0.015, 0.007, 0.005, 0.003, 0.003, 0.010, 0.020, 0.031, 0.058, 0.065, 0.056, 0.048, 0.042, 0.046, 0.036, 0.038, 0.038, 0.049, 0.087, 0.111, 0.090, 0.067, 0.044, 0.031', dishwasher.weekend_fractions)
+    assert_equal('1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0', dishwasher.monthly_multipliers)
   end
 
   def _check_refrigerator(hpxml_bldg, annual_kwh:, location:)
```
Comment (shorowit) [inline rulesets/tests/test_appliances.rb:402]: Can you add asserts for the refrigerator schedule coefficients?

**Review by shorowit — COMMENTED** [review]
Looks good, just two minor things.

File: rulesets/tests/test_occupancy.rb (hunk 1 above)
Comment (joseph-robertson) [reply rulesets/tests/test_occupancy.rb:9]: Oops. Good catch.

File: rulesets/tests/test_appliances.rb (hunk 2 above)
Comment (joseph-robertson) [reply rulesets/tests/test_appliances.rb:402]: Yes.

**Review by shorowit — APPROVED** [review]

**Pull request merged**